include/glaze/util/string_literal.hpp

```diff
@@ -30,7 +30,11 @@ namespace glz
       constexpr string_literal& operator=(const string_literal&) noexcept = default;
       constexpr string_literal& operator=(string_literal&&) noexcept = default;
 
-      constexpr string_literal(const char (&str)[N]) noexcept { std::copy_n(str, N, value); }
+      constexpr string_literal(const char (&str)[N]) noexcept {
+         for (size_t i = 0; i < N; ++i) {
```

Comment:
This is to avoid include of `<algorithm>` ?

Reply:
@sjanel, yes I'm slowly trying to simplify large header includes where they aren't really necessary.